### ISSUE TITLE
aria-hidden-focus reported

### ISSUE THREAD

**Issue by clapierre** [daisy/ace#253]
Ace Command Line version 1.1.1

I got this error today but the description of the error and linking to the KB is not helpful.

<p aria-hidden="true" epub:type="title" class="parttitle" id="part6"><a href="../Text/contents.xhtml#part6">RUPTURE OF RELATIONAL BONDS</a></p>
<h2 epub:type="title" class="parttitle1"><span class="sr-only">Rupture of Relational Bonds, </span>Violence, Affairs, and Divorce</h2>
Description from ACE:
Focusable content should have tabindex='-1' or be removed from the DOM
Learn more about: (no specific entry)

Not sure why they are trying to hide this link from the SR but then speak that same content minus the linkage. Very odd, will discuss this with the publisher but in any event is there anything to be done about the description and linkage to the KB?

**Comment by danielweck** [comment]
Hello Charles, this is an Axe message:
https://github.com/dequelabs/axe-core/blob/fa3c44b6e22f31a87df2a9e5686414a8fd1e429d/lib/checks/keyboard/focusable-not-tabbable.json#L8

The Javascript code that discovers such condition is:

https://github.com/dequelabs/axe-core/blob/fa3c44b6e22f31a87df2a9e5686414a8fd1e429d/lib/checks/keyboard/focusable-not-tabbable.js#L1-L30

Does this ring a bell in relation to the contents in your EPUB?

**Comment by gregoriopellegrino** [comment]
I get the same error.

We use aria-hidden="true" in the TOC file, for example for hiding landmarks, e.g.:

```
<nav epub:type="landmarks" hidden="hidden" aria-hidden="true">
      <ol>
        <li><a epub:type="titlepage" href="title.xhtml">Non tacciano le donne in assemblea</a></li>
        <li><a epub:type="bodymatter" href="introduction.xhtml">Introduzione</a></li>
        <li><a epub:type="bibliography" href="bibliography.xhtml">Bibliografia</a></li>
        <li><a epub:type="toc" href="toc.xhtml">Indice</a></li>
        <li><a epub:type="copyright-page" href="copyright.xhtml">Colophon</a></li>
      </ol>
</nav>
```

Do you think that `hidden` attribute and `aria-hidden` are redundant?

**Comment by jenstroeger** [comment]
@gregoriopellegrino, I think we needed the `hidden` to make sure that the `<nav>` element is not rendered by some readers. There was some issue here but I can’t recall the details 🤔

**Comment by gregoriopellegrino** [comment]
Yes, sure, I want to preserve the `hidden` attribute. My concern is about the `aria-hidden` attribute.

**Comment by jenstroeger** [comment]
Ah got it: you’re saying/asking that `aria-hidden` duplicates the meaning & intention of `hidden`?

**Comment by pkra** [comment]
> Do you think that hidden attribute and aria-hidden are redundant?

If `hidden` is used, then `aria-hidden=true` is redundant. It's even slightly dangerous - an element with `hidden` can easily become "unhidden" by accident (e.g., `<div hidden class="some class with display block">Hello</div>`) in which case `aria-hidden` would continue to hide the element from the accessibility tree.

**Comment by pkra** [comment]
Reference: https://w3c.github.io/html-aam/#att-hidden

**Comment by danielweck** [comment]
I am closing this issue as the responses seem to clarify the problem.
